<s> <user> hello guys ! i 'm having problems with a django application that uses a random forest <unka> to classify items . the error that i 'm receiving says : [ <unka'> object has no attribute <unka'_> ] and googling it leads to URL/http:/ URL/stackoverflow.com/ <unk#a-/> on the other hand , the exact line where the application throws the error is " <unka(._> )" and i 'm not using threads at all but i set <unk#a-=_> in the initialization of the <unka> so i wonder if this error could be related to <unka> ( it uses <unka> in the method " call " of the class parallel ). any idea ? can i modify the <unka_> variable in order avoid <unka> ? </s>
<s> <user> <unk#a@> please create a new stackoverflow question with a minimalistic reproduction script that generates its own test data . </s>
<s> <user> done ! URL/http:/ URL/stackoverflow.com/ <unk#a-/> </s>
<s> <user> answered . please include the full traceback to make it possible for us to fix the real cause of the problem . </s>
<s> <user> thanks <unka@> the error seems to be fixed after i set the <unka_> parameter to 1 . </s>
<s> <user> here is the complete traceback : <unk`> file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> DIR/core/ <unka/> <unka./> " in <unka_> line 111 . response = <unka(_> , <unka*_> , <unka*_> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> <unka/> <unka/> <unka./> " in <unka_> line 57 . return <unka(*_> , <unka*> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> <unka/> <unka/> <unka./> " in view line 69 . return <unka(.> , <unka*> , <unka*> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/_> <unka./> " in dispatch line 452 . response = <unka(._> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/_> <unka./> " in dispatch line <unk#> . response = <unka(> , <unka*> , <unka*> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/_> <unka./> " in handler line 50 . return <unka(*> , <unka*> ) file " DIR/home/ <unka/> <unka/> <unka/> <unka./> " in <unka_> line 70 . <unka_> = <unka(._> )) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> <unka./> " in <unka_> line 159 . return <unk#a(-.[]_> ) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> <unka/> <unka./> " in <unka_> line <unk#> . for i in <unka(_> )) file " DIR/home/ <unka/> DIR/env/ DIR/local/ DIR/lib/ DIR/python2.7/ DIR/site-packages/ <unka/> <unka/> <unka/> <unka./> " in <unka_> line <unk#> . <unka._> = <unka(_> ) file " DIR/usr/ DIR/lib/ DIR/python2.7/ <unka/> <unka./> " in <unka_> line <unk#> . <unka(._> , processes , <unka> , <unka> ) file " DIR/usr/ DIR/lib/ DIR/python2.7/ <unka/> <unka./> " in <unka_> line 136 . <unka(._> ) file " DIR/usr/ DIR/lib/ DIR/python2.7/ <unka/> <unka./> " in <unka_> line 199 . <unka(.> ) file " DIR/usr/ DIR/lib/ DIR/python2.7/ <unka/> <unka/> <unka./_> " in start line 73 . <unka.[_> ] = none exception type : attributeerror at <unka/> exception value : <unka'> object has no attribute <unka'_> <unk`> </s>
<s> <user> please edit the <unka> report instead . </s>
<s> <user> also quoting is done with backticks , not forward ticks :) </s>
<s> <user> sorry ! i was n't able to find the backticks in my spanish keyboard :/ </s>
<s> <user> i 've just added the traceback to stackoverflow ! </s>
